src/app/components/record-content/record-content.component.html

```diff
@@ -0,0 +1,16 @@
+<html>
+    <div class="container-fuild">
+        <div class="rContent">
+            <ul>
+                <li class="nd" *ngFor="let item of rounds">
+                    <div *ngIf="item.ganada == true" class="iCard green">
+                        Intentos: {{item.intentos}} Palabra: {{item.palabra}}
+                    </div>
+                    <div *ngIf="item.ganada == false" class="iCard red">
+                        Intentos: {{item.intentos}} Palabra: {{item.palabra}}
+                    </div>
```

Comment:
```suggestion
                    <div *ngIf="item.ganada else fail-option" class="iCard green">
                        Intentos: {{item.intentos}} Palabra: {{item.palabra}}
                    </div>
                    <ng-template #fail-option>
                       <div class="iCard red">
                           Intentos: {{item.intentos}} Palabra: {{item.palabra}}
                       </div>
                    </ng-template>
```

This is an option to do this.

And other valid option:

`<div *ngIf="someCheck()">`
`<div *ngIf="!someCheck()">`